src/components/shortcircuit-analysis-result.js

```diff
@@ -6,30 +6,127 @@
-import { PARAM_THEME } from '../utils/config-params';
-import { LIGHT_THEME } from '@gridsuite/commons-ui';
+import VirtualizedTable from './util/virtualized-table';
 
 const ShortCircuitAnalysisResult = ({ result }) => {
-    const selectedTheme = useSelector((state) => state[PARAM_THEME]);
+    const intl = useIntl();
 
     const shortCircuitNotif = useSelector((state) => state.shortCircuitNotif);
 
+    function flattenResult(shortcutAnalysisResult) {
+        const rows = [];
+        shortcutAnalysisResult?.faults?.forEach((f) => {
+            const fault = f.fault;
+            const limitViolations = f.limitViolations;
+            let firstLimitViolation;
+            if (limitViolations.length > 0) {
+                firstLimitViolation = {
+                    limitType: intl.formatMessage({
+                        id: limitViolations[0].limitType,
+                    }),
+                    limitMin:
+                        limitViolations[0].limitType ===
+                        'LOW_SHORT_CIRCUIT_CURRENT'
+                            ? limitViolations[0].limit
+                            : NaN,
+                    limitMax:
+                        limitViolations[0].limitType ===
+                        'HIGH_SHORT_CIRCUIT_CURRENT'
+                            ? limitViolations[0].limit
+                            : NaN,
+                    limitName: limitViolations[0].limitName,
+                    current: limitViolations[0].value,
+                };
+            }
+            rows.push({
+                faultId: fault.id,
+                elementId: fault.elementId,
+                faultType: intl.formatMessage({ id: fault.faultType }),
+                shortCircuitPower: f.shortCircuitPower,
+                current: f.current,
+                ...firstLimitViolation,
+            });
+            limitViolations.slice(1).forEach((lv) => {
+                rows.push({
+                    limitType: lv.limitType,
+                    limit: lv.limit,
+                    limitName: lv.limitName,
+                    value: lv.value,
+                });
+            });
+            const feederResults = f.feederResults;
+            feederResults.forEach((fr) => {
+                rows.push({
+                    connectableId: fr.connectableId,
+                    current: fr.current,
+                });
+            });
+        });
+        return rows;
+    }
+
     function renderResult() {
+        const rows = flattenResult(result);
         return (
             result &&
             shortCircuitNotif && (
-                <ReactJson
-                    src={result}
-                    onEdit={false}
-                    onAdd={false}
-                    onDelete={false}
-                    theme={
-                        selectedTheme === LIGHT_THEME
-                            ? 'rjv-default'
-                            : 'monokai'
-                    }
+                <VirtualizedTable
```

Comment:
In the tab view and in the loadflow results, numerics are justified left. 
In the security analysis results, numerics are justified right as you did.
Maybe we should homogenize this ?

Reply:
Maybe it would be better to homogenize this in all tables in an other PR.